### PR TITLE
CHE-9720: Checkout ssh-extension to tag with extension

### PR DESCRIPTION
**Pull request by vinokurig**
### What does this PR do?
Checkout `che-theia-ssh-plugin` to tag with extension. Master of the extension's repository is going to contain theia plugin instead of extension.

### What issues does this PR fix or reference?
#9720 

#### Release Notes

N/A

#### Docs PR

N/A

### CODE REVIEW

File: dockerfiles/theia/src/extensions.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
             "name": "che-theia-ssh-extension",
             "source": "https://github.com/eclipse/che-theia-ssh-plugin",
             "folder": "ssh-extension",
-            "checkoutTo": "master",
+            "checkoutTo": "0.0.0",
```
Comment (benoitf) [inline dockerfiles/theia/src/extensions.json]: could we have as tag like 0.0.1 ?

Comment (vinokurig) [reply dockerfiles/theia/src/extensions.json]: Sure but let's set the new version in the extension: https://github.com/eclipse/che-theia-ssh-plugin/pull/10

Comment (vinokurig) [reply dockerfiles/theia/src/extensions.json]: done

**Comment by benoitf** [comment]
ci-build

**Review by benoitf — APPROVED** [review]

**Comment by vinokurig** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/895/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/895//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11405
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by vinokurig** [comment]
I am going on PTO, will follow it in 3 weeks

**Comment by benoitf** [comment]
let's merge it

**Comment by benoitf** [comment]
it's approved

**Pull request merged**